binding-library/java/src/main/java/com/microsoft/azure/functions/rabbitmq/annotation/RabbitMQTrigger.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+ 
+package com.microsoft.azure.functions.rabbitmq.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+
+/**
+ * <p>Place this on a parameter whose value would come from RabbitMQ and causing the method to run when a new event arrives.
+ * The parameter can be one of the following</p>
+ *
+ * <ul>
+ *     <li>A String java native type</li>
+ *     <li>Any POJO type</li>
+ * </ul>
+ */
```

Comment:
nit: same here. Other basic types are supported as well.

Reply:
I don't think directly binding to byte[], int works for even for dotnet today. 

My understanding was that for in-proc, it would be string, POCO or BasicDeliverEventArgs and for out-of-proc, they would be normal strings or stringified json (except in java, json can be converted to the appropriate POJO as well). Am I missing something else ?

Reply:
in dotnet function, user should be able to bind to byte[] as that is the type of the body event is is delivered. If not, please fix this.

For java - as its a typed language - users can request any type - java worker is responsible for converting input data to the type requested in the method signature.

Reply:
Ah, okay.

The body of event is delivered as byte[] but wrapped as BasicDeliverEventArgs. I was assuming that's why binding to BasicDeliverEventArgs was introduced. I think this probably means a gap that needs to be addressed where we support binding to both byte[] and BasicDeliverEventArgs, right ?